homeassistant/components/camera/neato.py

```diff
@@ -40,6 +40,8 @@ def __init__(self, hass, robot):
         self.neato = hass.data[NEATO_LOGIN]
         self._image_url = None
         self._image = None
+        self._unique_id = '{}-{}'.format(
```

Comment:
Is the name necessary here? If the serial will suffice (i.e., be sufficiently unique for each device), just use that.

Reply:
@bachya I honestly was not sure, I assumed that I could not use just the same robot serial for all components.  If that is the case then I can update this PR to include the serial for each component.